src/frontend/app/App.ts

```diff
@@ -69,13 +70,14 @@ export class App {
     FrontstageManager.addFrontstageProvider(new SnapshotSelectFrontstage());
   }
 
-  public static async initializeOidc() {
-    const scope = "openid email profile organization imodelhub context-registry-service:read-only product-settings-service urlps-third-party offline_access";
-    const clientId = Config.App.getString("IMJS_ELECTRON_TEST_CLIENT_ID");
+  public static getAuthConfig(): NativeAppAuthorizationConfiguration {
     const redirectUri = Config.App.getString("IMJS_ELECTRON_TEST_REDIRECT_URI");
-    const oidcConfiguration: DesktopAuthorizationClientConfiguration = { clientId, redirectUri, scope };
-    const desktopClient = new DesktopAuthorizationClient(oidcConfiguration);
-    await desktopClient.initialize(new ClientRequestContext());
-    IModelApp.authorizationClient = desktopClient;
+    const scope = "openid email profile organization imodelhub context-registry-service:read-only product-settings-service urlps-third-party offline_access";
+
+    return {
+      clientId: "imodeljs-electron-test",
```

Comment:
I don't think we should hardcode the clientId in the app itself. This should be changed by all new devs when they go through the registration process.

Reply:
Yep, good catch

Reply:
@kckst8 pointed out that this isn't actually going to work. You're reading from `Config.App`, which isn't loaded until after `ElectronApp.startup` - a catch 22. In [this PR](https://github.com/imodeljs/imodeljs/pull/1105) I provide a way to supply the authConfig from the backend. I think you should wait for that PR to complete and make the same changes @kckst8 makes in the iTwin desktop app.

Reply:
Ok. That is probably why I hardcoded the scopes and clientId.

Reply:
Could I use `process.env` instead of `Config.App`?

Reply:
You could technically do that since we load all environment variables into the frontend as `process.env` and that's what `Config.App` is using.

Just talked to @kabentley and we're going to backport his change to 2.14.1 in a few minutes.

We're going to have a larger conversation around how to handle configuration better in Desktop since there's differences between backend and frontend.

Reply:
Ok. This PR will wait for that backport and implement that method.

Reply:
Really, i don't think anything about "authConfig" should be configurable at runtime. It should all be specified by the application at build time. @kckst8